WordPress/src/main/java/org/wordpress/android/util/config/BloggingPromptsListFeatureConfig.kt

```diff
@@ -0,0 +1,16 @@
+package org.wordpress.android.util.config
+
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.annotation.FeatureInDevelopment
+import javax.inject.Inject
+
+@FeatureInDevelopment
+class BloggingPromptsListFeatureConfig
+@Inject constructor(appConfig: AppConfig) : FeatureConfig(
+        appConfig,
+        BuildConfig.BLOGGING_PROMPTS_LIST
```

Comment:
I was thinking about doing that when the feature is minimally complete, so we don't have issues with users on different versions (partial implementation) seeing that partial feature when we create and enable a remote flag in the future.

For instance: Having the `remoteField` in a partial state would cause that since all versions that read that remote flag would see it as "enabled" regardless of the feature being incomplete or not in the currently installed version.